master/custom/builders.py

```diff
@@ -185,6 +187,10 @@ def get_builders(settings):
         ("AMD64 Cygwin64 on Windows 10", "bray-win10-cygwin-amd64", UnixBuild, UNSTABLE),
         ("PPC64 AIX", "edelsohn-aix-ppc64", AIXBuild, UNSTABLE),
         ("PPC64 AIX XLC", "edelsohn-aix-ppc64", AIXBuildWithXLC, UNSTABLE),
+
+        # Windows/arm64
+        ("ARM64 Windows", "linaro-win-arm64", WindowsARM64Build, STABLE),
+        ("ARM64 Windows Non-Debug", "linaro-win-arm64", WindowsARM64ReleaseBuild, STABLE),
```

Comment:
Please start with UNSTABLE. I prefer to wait until it is reliable before marking them as STABLE.

Reply:
Sure. I've added a new patch to mark the workers as UNSTABLE